tests/test_browser.py

```diff
@@ -605,7 +628,7 @@ def test_sdl_image(self):
     open(os.path.join(self.get_dir(), 'sdl_image.c'), 'w').write(self.with_report_result(open(path_from_root('tests', 'sdl_image.c')).read()))
 
     for mem in [0, 1]:
-      for dest, dirname, basename in [('screenshot.jpg',                        '/',       'screenshot.jpg'),
+      for dest, dirname, basename in [('screenshot.jpg', '/', 'screenshot.jpg'),
```

Comment:
i slightly like the older indentation, which lines things up with the line below. but no strong feelings either way.

Reply:
I do too.  Especially for larger structs.   I can't seem to find any way to allow these in flake8 though, aside from annotating the individual lines.. which I'm not a fan of either.